.github/workflows/cypress.yml

```
@@ -34,6 +35,23 @@ jobs:
                     echo "branch=${{ github.ref_name }}" >> $GITHUB_OUTPUT
                   fi
 
+            - name: Configure AWS credentials
+              uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)